Angular/.ci/test_unix-any.sh

```diff
@@ -44,13 +45,13 @@ if [ $(OS_Is_Run_Simulated) -eq 0 ]; then
         I18N_Simulate_Testing
         return 0
 else
-        ___browser="$(type -p google-chrome)"
+        ___browser="$(type -p chromium)"
```

Comment:
No reason to change this. Revert it.

Reply:
No longer spotted in the new force-commit.

Reply:
concur.